retrodep/workingtree.go

```diff
@@ -272,3 +277,24 @@ func (wt *anyWorkingTree) StripImportComment(path string, w io.Writer) (bool, er
 
 	return changed, nil
 }
+
+// Diff writes output to stdout from 'diff -Nu' comparing the
+// path within the working tree with the localFile.
+func (wt *anyWorkingTree) Diff(out io.Writer, path, localFile string) error {
+	if path == "" {
+		path = "/dev/null"
+	} else if path[0] != '/' {
+		path = filepath.Join(wt.Dir, path)
+	}
+
+	p := execCommand("diff", "-Nu", path, localFile)
+	p.Stdout = out
+	err := p.Run()
+
+	// Ignore the exit status.
```

Comment:
If this is wanting to ignore an exit status of 0 or 1 then it's discarding more than that.  More than one means an unexpected problem.  And then there's signals too.  If ExitCode isn't 0 or 1 then maybe the error should ripple up?

Reply:
Thanks.

Reply:
Should be fixed now.

Reply:
Not using .ExitCode for now as that's brand new in Go 1.12.